src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -0,0 +1,422 @@
+                  "device_database",
+                  html`<a href="https://devices.zwave-js.io/" target="_blank"
+                    >${this.hass.localize(
+                      "ui.panel.config.zwave_js.node_config.zwave_js_device_database"
+                    )}</a
+                  >`
+                )}
+              </em>
+            </p>
+          </div>
+          ${this._config
+            ? html`
+                ${Object.entries(this._config).map(
+                  ([id, item]) => html` <ha-card
+                    class="content config-item"
+                    .configId=${id}
+                  >
+                    <div class="card-content">
+                      ${this._generateConfigBox(id, item)}
+                    </div>
+                  </ha-card>`
+                )}
+              `
+            : ``}
+        </ha-config-section>
+      </hass-tabs-subpage>
+    `;
+  }
+
+  private _generateConfigBox(id, item): TemplateResult {
+    const labelAndDescription = html`
+      <div class="config-label">
+        <b>${item.metadata.label}</b><br />
+        <span class="secondary">
+          ${item.metadata.description}
+          ${item.metadata.description !== null && !item.metadata.writeable
+            ? html`<br />`
+            : ""}
+          ${!item.metadata.writeable
+            ? html`<em
+                >${this.hass.localize(
+                  "ui.panel.config.zwave_js.node_config.parameter_is_read_only"
+                )}</em
+              >`
+            : ""}
+        </span>
+      </div>
+    `;
+
+    // Numeric entries with a min value of 0 and max of 1 are considered boolean
+    if (
+      (item.configuration_value_type === "range" &&
+        item.metadata.min === 0 &&
+        item.metadata.max === 1) ||
+      this._isEnumeratedBool(item)
+    ) {
+      return html`
+        <div class="flex">
+          ${labelAndDescription}
+          <div class="toggle">
+            <ha-switch
+              .property=${item.property}
+              .propertyKey=${item.property_key}
+              .checked=${item.value === 1}
+              .key=${id}
+              @change=${this._switchToggled}
+              .disabled=${!item.metadata.writeable}
+            ></ha-switch>
+          </div>
+        </div>
+      `;
+    }
+
+    if (item.configuration_value_type === "range") {
+      return html`${labelAndDescription}
+        <paper-input
+          type="number"
+          .value=${item.value}
+          .min=${item.metadata.min}
+          .max=${item.metadata.max}
+          .property=${item.property}
+          .propertyKey=${item.property_key}
+          .key=${id}
+          .disabled=${!item.metadata.writeable}
+          @value-changed=${this._numericInputChanged}
+        >
+        </paper-input> `;
+    }
+
+    if (item.configuration_value_type === "enumerated") {
+      return html`
+        ${labelAndDescription}
```

Comment:
Shouldn't this be inside the `div.flex`?

Reply:
flex div was to get the dropdown to take up the width of its parent, was having difficulty getting it to fill its parent otherwise